plugins/teststeps/fileupload/awsFileUpload.go

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package fileupload
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/facebookincubator/contest/pkg/event"
+	"github.com/facebookincubator/contest/pkg/event/testevent"
+	"github.com/facebookincubator/contest/pkg/target"
+	"github.com/facebookincubator/contest/pkg/test"
+	"github.com/facebookincubator/contest/pkg/xcontext"
+	"github.com/facebookincubator/contest/plugins/teststeps"
+	"github.com/prometheus/common/log"
+)
+
+// Define some constants for uploading things into a S3 bucket
+const (
+	S3_REGION = "" // your aws server region
+	S3_BUCKET = "" // your aws bucket
+	S3_PATH   = "" // your path where to upload the file
```

Comment:
Shouldn't this be parameters of the plugin? Do I need to change this file to use this plugin?
(Or maybe the plugin should read this from some file?)